sdk/extensions/Microsoft.Extensions.Azure/src/AzureEventSourceLogForwarder.cs

```diff
@@ -49,6 +47,8 @@ private void LogEvent(EventWrittenEventArgs eventData)
                 return;
             }
 
+            // The same logic is used in the Azure.Monitor.OpenTelemetry.AspNetCore.
```

Comment:
I'm not comfortable with this as a strategy and would like to talk this over.

Reply:
Th reasons not to add a dependency to Mictosoft.Exensions.Logging are

- copy-paste is tiny
- no additional dependencies (such as Identity coming via Extensions)
- In future we can optimize EventSource -> OTel format translation without using ILogger, so in the long term distro would need a different forwarder impl anyway.

I hope it explains that copying a few lines of code is the least of evils in this case.

Reply:
per offline discussion, tiny copy-paste is not a problem. Deviation between `Microsoft.Extensions.Azure` log forwarder and distro's forwarder is inevitable and even desirable, so I removed the note here.